apps/static/dance/ai/model/cached-spacy-background-map.json

```diff
@@ -185,7 +185,37 @@
     "squares",
     "squiggles",
     "starburst",
+    "higher_power"
```

Comment:
These lists don't look quite correct. When I run your script, my lists look different for some reason.